docs/13-walkthroughs/6-publish-and-deploy.md

```diff
@@ -1,101 +1,54 @@
-# Using OCI to Store & Deploy Zarf Packages
+# Using OCI to Store & Deploy Zarf Packages 
 
 ## Introduction
 
-In this walkthrough, we are going to run through how to publish a Zarf package to an [OCI](https://github.com/opencontainers/image-spec) compliant registry, allowing end users to pull and deploy packages without needing to build locally, or transfer the package to their environment.
+In this walkthrough, we are going to run through how to publish a Zarf package to an OCI compliant registry, allowing end users to pull and deploy packages without needing to build locally, or transfer the package to their environment.
+
+## System Requirements
+
+- Internet access to download any images or access to Docker Hub
+- A method to run a registry container, if wishing to run a local registry for testing.
 
 ## Prerequisites
 
 For following along locally, please ensure the following prerequisites are met:
 
 1. Zarf binary installed on your `$PATH`: ([Install Instructions](../3-getting-started.md#installing-zarf))
 2. Access to a [Registry supporting the OCI Distribution Spec](https://oras.land/implementors/#registries-supporting-oci-artifacts), this walkthrough will be using Docker Hub
+2. [Initialize a cluster](./1-initializing-a-k8s-cluster.md).
 
-## Setup
+## Walkthrough
+[![Using OCI to Store & Deploy Zarf Packages Video on YouTube](../.images/walkthroughs/publish_and_deploy_thumbnail.png)](https://www.youtube.com/watch?v=QKxgJnC_37Y "Using OCI to Store & Deploy Zarf Packages")
```

Comment:
Why are we embedding a link w/ a thumbnail and not a direct Youtube embed here? Example: https://github.com/Noxsios/zola-theme-terminimal/blob/271ffecddef428b43c66b5bf592cf41016b73fb4/templates/shortcodes/youtube.html

Reply:
Can make this a react component, and use across files.  The ANSI colorings can also be react components due to their repetitive style declarations.

Reply:
We've been using this style with the walkthroughs so far.  I'll defer to @Racer159 on how he'd like these done.  If we switch it up here, we'll have to backport changes to the walkthroughs.

Reply:
Yeah we could do it this way for now, and then refactor it all in one fell swoop if we wanted - I agree an embed is prolly better overall but can also wait a little bit.